app/__init__.py

```diff
@@ -0,0 +1,9 @@
+import config
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+app.config.from_object(config.config_environments['development'])
+app.config['SECRET_KEY'] = "123bucketlistapp"
```

Comment:
It's best practice to put your secret key as an environment variable then use `os.env` to initialize it in your application.